flink-core/src/main/java/org/apache/flink/core/fs/FileSystem.java

```diff
@@ -255,14 +252,47 @@
 	 * @param config the configuration from where to fetch the parameter.
 	 */
 	public static void initialize(Configuration config) throws IllegalConfigurationException {
```

Comment:
`@Deprecated`?

Reply:
I did not deprecate it because it is also used internally in the class, but I have an idea how to change and improve it.